queries/go/highlights.scm

```diff
@@ -0,0 +1,138 @@
+;; Forked from tree-sitter-go 
+;; Copyright (c) 2014 Max Brunsfeld (The MIT License)
+
+;;
+; Function calls
+
+(call_expression
+  function: (identifier) @function)
+
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+; Function definitions
+
+(function_declaration
+  name: (identifier) @function)
+
+(method_declaration
+  name: (field_identifier) @function.method)
+
+; Identifiers
+
+(type_identifier) @type
+(field_identifier) @property
+(identifier) @variable
+
+(parameter_declaration (identifier) @parameter)
+(variadic_parameter_declaration (identifier) @parameter)
+
+((identifier) @constant
+ (eq? @constant "_")) 
+
+;; TODO: This variable is so prominent in Go
+;; it deserves a special hightlight. Which?
+;((identifier) @constant
+ ;(eq? @constant "err")) 
```

Comment:
For this, is using `constant` or `constant.builtin` the kind of solution you are searching for ?
Otherwise, adding a special `variable.builtin` can be a good idea too.

Reply:
Looking a bit at it. I found it confusing for it to have a different. I will just omit that highlight.